src/components/Checkout/OrderSummary.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import {
+	Heading,
+	VStack,
+	SimpleGrid,
+	GridItem,
+	Text,
+	Button,
+	Divider,
+} from '@chakra-ui/react';
+import { colors } from '../../util/constants';
+
+import { useStripe, useElements } from '@stripe/react-stripe-js';
+
+const OrderSummary = () => {
+	const stripe = useStripe();
+	const elements = useElements();
+
+	const [message, setMessage] = useState(null);
+	const [isLoading, setIsLoading] = useState(false);
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		if (!stripe || !elements) {
+			// Stripe.js has not yet loaded.
+			// Make sure to disable form submission until Stripe.js has loaded.
+			return;
+		}
+
+		setIsLoading(true);
+
+		const { error } = await stripe.confirmPayment({
+			elements,
+			confirmParams: {
+				return_url: 'http://localhost:3000/checkout',
```

Comment:
Use the `REACT_APP_PROXY` here instead of hard-coding. It won't be `http://localhost:3000/` on Netlify and for other developers working on this locally.

Reply:
If you decide to go about this, add it to the documentation as well in `README.md`.

Reply:
done